packages/view3d/src/View3D.ts

```diff
@@ -123,6 +124,12 @@ export interface View3DOptions {
   quickLook: boolean | Partial<QuickLookSessionOptions>;
   arPriority: Array<ValueOf<typeof AR_SESSION_TYPE>>;
 
+  // Post-processing
+  ssr: boolean | Partial<SsrOptions>;
+  ssao: boolean | Partial<SsaoOptions>;
+  bloom: boolean | Partial<BloomOptions>;
+  dof: boolean | Partial<DoFOptions>;
```

Comment:
Try making these not as options of View3D, but as plugins.
That's for the tree-shaking, as post-processing is not the thing that everyone will use.

Reply:
You mean, like plugin as egjs-flicking?
https://naver.github.io/egjs-flicking/Plugins

Reply:
Right, View3D also has plugins in it.
- https://github.com/naver/egjs-view3d/tree/main/packages/view3d/src/plugin
- https://naver.github.io/egjs-view3d/docs/plugins

Reply:
Good. Also i'll move to plugin docs too. :)